### DISCUSSION TITLE
Operon - command line editing multiple tags

### DISCUSSION THREAD

**Discussion by davidSchlosser** [quodlibet/quodlibet#4258] [Q&A]
My understanding is that Operon only supports changing multiple tags in a song file using the `edit` command with an interactive editor, defaulting to Nano. Is there an existing way to provide a list to reset all the tags (like `edit` gets from the editor) to Operon in the command line, or via a file?

I'd like to be able to do something like the following but with many more tags:

`$operon reset='title=A Day In The Life\nartist=The Beatles\ngenre=Pop' 01-Beatles.ogg`

Thanks.

**Comment by bassstorm** [comment]
I personally ended up with using more low-level tools in my migration scripts, e.g. `metaflac`, `vorbiscomment`.
